data/syncer/userAccountsSyncer.go

```diff
@@ -157,7 +156,9 @@ func (u *userAccountsSyncer) syncDataTrie(rootHash []byte, ssh data.SyncStatisti
 		return err
```

Comment:
Not need to call u.throttler.EndProcessing() on return err cases in this method ?